.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,7 @@
+-   id: pycodestyle
+    name: pycodestyle
+    description: 'pycodestyle (formerly called pep8) - Python style guide checker'
```

Comment:
This should just be `pycodestyle - Python style guide checker` or `Python style guide checker`

Reply:
Thanks for your comment. I changed it.